ArcGISRuntimeSDKQt_QMLSamples/Geometry/ConvexHull/ConvexHull.qml

```diff
@@ -0,0 +1,127 @@
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.8
+import QtQuick.Layouts 1.11
+import QtQuick.Controls 2.6
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            BasemapTopographic {}
+        }
+
+        GraphicsOverlay {
+            id: graphicsOverlay
+
+            Graphic {
+                id: inputsGraphic
+                SimpleMarkerSymbol {
+                    id: markerSymbol
+                    style: Enums.SimpleMarkerSymbolStyleCircle
+                    size: 10
+                    color: "red"
+                }
+            }
+
+            Graphic {
+                id: convexHullGraphic
+            }
+        }
+
+        SimpleFillSymbol {
+            id: fillSymbol
+            style: Enums.SimpleFillSymbolStyleNull
+            color: "transparent"
+            SimpleLineSymbol {
+                id: lineSymbol
+                style: Enums.SimpleLineSymbolStyleSolid
+                width: 3
+                color: "blue"
+            }
+        }
+
+        // stores clicked points
+        PointCollection {
+            id: inputs
+            spatialReference: mapView.spatialReference
+        }
+
+        MultipointBuilder {
+            id: multipointBuilder
+            spatialReference: mapView.spatialReference
+        }
+
+        onMouseClicked: {
+            var clickedPoint = mapView.screenToLocation(mouse.x, mouse.y);
+            inputsGraphic.geometry = clickedPoint;
+            inputs.addPoint(clickedPoint);
+            multipointBuilder.points = inputs;
+            inputsGraphic.geometry = multipointBuilder.geometry;
+            console.log(inputs.size);
+        }
+
+        RowLayout {
+            anchors {
+                left: parent.left
+                top: parent.top
+            }
+
+            Button {
+                width: implicitWidth
+                height: implicitHeight
+                text: "Convex Hull"
+
+                // display the convex hull
+                onClicked: {
+                    var convHull = GeometryEngine.convexHull(multipointBuilder.geometry);
+                    if (convHull.geometryType === Enums.GeometryTypePoint) {
+                        convexHullGraphic.symbol = markerSymbol;
+                    } else if (convHull.geometryType === Enums.GeometryTypePolyline) {
+                        convexHullGraphic.symbol = lineSymbol;
+                        console.log("polyline");
+                    } else if (convHull.geometryType === Enums.GeometryTypePolygon) {
+                        convexHullGraphic.symbol = fillSymbol;
+                        console.log("polygon");
+                    }
+
+                    convexHullGraphic.geometry = convHull;
+                }
+            }
+
+            Button {
+                width: implicitWidth
```

Comment:
Does the width and height need to be set if they take the values of the implict width and height?

If you refer to the `RowLayout` documentation you can see that `RowLayout` injects `Layout` properties into its children.

Try adding `Layout.fillWidth: true` to this and to the other button instead if you are trying to get them to be of the same size?